packages/scss/src/components/_action-icon.scss

```diff
@@ -36,19 +17,25 @@
 	cursor: pointer;
 
 	& + & {
-		margin-left: .1rem;
+		margin-left: 2px;
```

Comment:
where this number come from ?

Reply:
It's the margin between 2 action-icons. 
As it was related to a px value for outline effect, it seemed fair to harmonize it in px everywhere.